google/cloud/bigtable/examples/BUILD

```diff
@@ -39,3 +39,9 @@ cc_binary(
     srcs = ["bigtable_samples_instance_admin.cc"],
     deps = ["//google/cloud/bigtable:bigtable_client"],
 )
+
+cc_binary(
+    name = "bigtable_samples_instance_admin_ext",
```

Comment:
The name is unfortunate.  What about renaming the old one to `bigtable_instance_admin_snippets`?

Reply:
We can do that in a future PR btw, just remember to open a bug in GitHub.

Reply:
#881 created for this purpose.